src/components/Header.tsx

```diff
@@ -1,7 +1,9 @@
 export default function Header() {
   return (
-    <div className="flex h-24 flex-row items-center justify-center">
```

Comment:
add a background color so we can differentiate it from the other components for now